README.md

```diff
@@ -35,7 +35,7 @@
 
 > Describe your project in 1 or 2 sentences.
 
-**[Book store]** "Book store" is an MVP version of it that allows you to:
+**[Book store using react-redux]** "Book store" is an MVP version of it that allows you to:
```

Comment:
- [ ] Please 🙏🏽 correct the title of the project, it is Book Store, not Math magicians for a professional README file.
- [ ] This is not the To-Do List app, please actualize the links on Your README file to have professional documentation. 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/216224258-7e4c664a-a704-44a0-8894-45d7d6577bd4.png)


- [ ] We are not using webpack for this project, please make sure to actualize each section on your README for a professional README file.

![image](https://user-images.githubusercontent.com/58642949/216224036-46376c52-2a39-44d5-bdec-5c8f18b5dfa9.png)

- [ ] Kindly add the LICENSE file to your repo and make sure that the MIT link works as expected.